engine/class_modules/warlock/sc_warlock_affliction.cpp

```diff
@@ -395,6 +395,29 @@ namespace warlock
           p()->buffs.active_uas->decrement( 1 );
 
           affliction_spell_t::last_tick( d );
+
+          if ( p()->azerite_essence.conflict_and_strife.enabled() &&
+            p()->azerite_essence.conflict_and_strife.is_major() )
+          {
+            warlock_td_t* target_data = td( d->target );
+
+            bool any_ticking = false;
+            for (int i = 0; i < MAX_UAS; i++)
```

Comment:
You could use a range base for loop here.
for (auto& ua_dot : target_data->dots_unstable_affliction )